main/glossary/README.md

```diff
@@ -112,13 +112,9 @@ Destroy all digital assets in a payment. See [`issuer.burn(payment, optAmount)`]
 
 ## Comparable
 
-A *passable* is something that can be marshalled. A *comparable* is a
-passable whose leaves contain no promises. Two comparables can be
-synchronously compared for structural equivalence.
-
-A comparable is a JavaScript object containing no promises, and can
-thus be locally compared for equality with another object. If either object
-contains Promises, equality is indeterminable. If both are fulfilled down
+A *comparable* is a [passable](#passable) whose leaves contain no promises. Two 
+comparables can be synchronously compared for structural equivalence. If either passable
+object contains Promises, equality is indeterminable. If both are fulfilled down
```

Comment:
I think this can be made a bit clearer. How about 
"A *comparable* is a [passable](#passable) which does not contain any Promises, It can be locally and synchronously compared to another object for equality, including structural equivalence. But if either passable contains Promises, equality cannot be determined. If both objects are fulfilled down to Presences and local state, then either they're the same all the way down, or they represent different objects."